stable/luigi/templates/deployment.yaml

```diff
@@ -0,0 +1,71 @@
+apiVersion: extensions/v1beta1
```

Comment:
For the deployment apiversion can you use `apps/v1beta2`? This extensions apiversion may go away soon due to where it is in the deprecation cycle.